src/tests/SeqTests.fs

```diff
@@ -362,7 +357,7 @@ let ``Seq.sortDescending works``() =
 let ``Seq.sortBy works``() =
     let xs = [3.; 1.; 4.; 2.]
     let ys = xs |> Seq.sortBy (fun x -> -x)
-    sumFirstTwo ys
+    sumFirstSeq ys 2
```

Comment:
This is a bit tricky, because `sumFirstSeq` is supposed to have the same function as `sumFirstTwo`. I copied these tests almost literally from FunScript and they can surely be improved. But for the sake of consistence, I'll keep as they are and add extra assertions wherever needed.